src/main/features/claim/views/amount.njk

```diff
@@ -83,7 +83,10 @@
               {% if canAddMoreRows %}
                 {{ linkButton('Add another row', 'action[addRow]', 'button button-secondary') }}
               {% endif %}
-              {{ linkButton('Calculate total', 'action[calculate]', 'link-button calculate') }}
+              <noscript>
+                {{ linkButton('Calculate total', 'action[calculate]', 'link-button calculate') }}
```

Comment:
where is the non-js action being called?

Reply:
Clicking this button refreshes the page when JS is disabled.